activerecord/test/cases/base_test.rb

```diff
@@ -968,6 +968,14 @@ def test_previously_new_record_returns_boolean
     assert_equal false, Topic.find(1).previously_new_record?
   end
 
+  def test_previously_new_record_on_destroyed_record
+    topic = Topic.create
+    assert topic.previously_new_record?
+
+    topic.destroy
+    assert_not topic.previously_new_record?
```

Comment:
Even though I don't find `assert_not_predicate` name particularly friendly I think it pays off with a better error message :) 
```suggestion
    assert_not_predicate topic, :previously_new_record?
```